source/shared/product/_sidebar.html.erb

```diff
@@ -1,7 +1,6 @@
 <% menu_items = [
   { title: "Connect to GovWifi", link: "/about-govwifi/connect-to-govwifi/" },
   { title: "What is GovWifi?", link: "/about-govwifi/" },
-  { title: "Who GovWifi is for", link: "/about-govwifi/who-govwifi-is-for/" },
```

Comment:
The screenshot still has this in it. Is this meant to be removed?

Reply:
Ah yes, sorry. Old screenshot.